### ISSUE TITLE
Add docs about VIRTUAL_ENV.

### ISSUE THREAD

**Issue by huxuan** [pdm-project/pdm#2856]
## Is your feature/enhancement proposal related to a problem? Please describe.

When integrating readthedocs, I did not find the usage of setting the path of virtual environment until I looked through the `.readthedocs.yaml` configuration in pdm repository.

## Describe the solution you'd like

We should add docs for VIRTUAL_ENV or even a whole list of environment variables.

**Comment by frostming** [comment]
It is more related to readthedocs, and stackoverflow may be a better place for this.

**Comment by huxuan** [comment]
I have to admit I searched a lot on the Internet and did not find any useful reference, the keywords I used are somthing like `pdm readthedocs virtual environments`. I even asked ChatGPT but still get no easy solution. Until then I realized pdm project itself is integrated with ReadTheDocs, too. That is the perfect solution I have ever found. Maybe we can add a section `Use PDM in ReadTheDocs` just like `Use PDM in Continuous Integration` in the `Advanced Usage`?
